### DISCUSSION TITLE
Clarification regarding https://github.com/python-poetry/poetry/issues/4325

### DISCUSSION THREAD

**Discussion by meln5674** [python-poetry/poetry#8982] [Q&A]
Hello,

I recently encountered an issue that I believe to be the same as https://github.com/python-poetry/poetry/issues/4325 .

Is the first post still the position of the maintainers?
If so, why is this issue not closed as wontfix? Is it intended that this one day be supported?
Where is this fact documented, along with what is expected/required to correctly install a package over git?

Thank you for your time.

**Comment by radoering** [comment]
> Is the first post still the position of the maintainers?

You cannot know if it ever was because no maintainer responded (although the comment is from a very experienced contributor and often corresponds to the position of the maintainers).

> If so, why is this issue not closed as wontfix?

Because maintainer time is limited and we are not able to take a closer look at each issue.

> Is it intended that this one day be supported?

I commented the issue. I don't think that any regular contributor is interested in supporting this but I assume we would accept a PR if the issue is important enough for someone to provide one.

> Where is this fact documented, along with what is expected/required to correctly install a package over git?

If you don't find anything in the docs, it might not be documented.
